src/components/Content/Content.tsx

```diff
@@ -174,50 +111,52 @@ const Content = (props: ContentProps): ReactElement => {
 		)
 	};
 
+	/**
+	 * Generates the sections that will be iterated to render the fields
+	 * as they're positioned. If no sections are provided it will return
+	 * a one-single column section using the field's name or colum if defined.
+	 * @returns the sections that contains the names of the fields
+	 */
+	const sectionsToRender = useMemo(() => {
+		const newSections = []
+
+		if (!sections) {
+			fields.forEach(field => {
+				const fieldName = field?.column ? field.column : field.name;
+				newSections.push([[ fieldName ]])
+			});
+
+			return newSections;
+		}
+
+		return sections;
+	}, [sections]);
+
 	return (
 		<MainWrapper>
 			<TitleWrapper>
 				<Title>{title}</Title>
-				<ButtonsWrapper canShowMore={canShowMore && values}>
-					{!values
-						? onEdit && (
-							<Button
-								color="teal"
-								variant="icon"
-								mIcon={AddIcon}
-								onClick={onEdit}
-							></Button>
-						)
-						: onEdit && (
-							<Button
-								color="gray"
-								variant="icon"
-								mIcon={EditIcon}
-								onClick={onEdit}
-							></Button>
-						)}
-					{canShowMore && values && (
-						<Button
-							color="teal"
-							variant="text"
-							label={showMore ? "Less Details" : "More Details"}
-							onClick={showDetails}
-						></Button>
-					)}
+				<ButtonsWrapper>
+					{buttons.map((button, idx) => (
+						<Button key={`${button.label}-${idx}`} {...button} />
+					))}
 				</ButtonsWrapper>
 			</TitleWrapper>
-			{values && (
-				<ColumnsContainer className={`${showMore ? "expanded" : "collapsed"}`} ref={columnsContainerRef}>
-					<ContentColumn ref={leftColumnRef}>
-						{renderColumn(leftColumn, "left")}
-					</ContentColumn>
-					{rightColumn && rightColumn.length > 0 && (
-						<ContentColumn>
-							{renderColumn(rightColumn, "right")}
-						</ContentColumn>
-					)}
-				</ColumnsContainer>
-			)}
+			<div>
+				{data && sectionsToRender.map((section, idx) => {
+					if (section?.length >= 3) {
```

Comment:
I believe this error could be thrown in the sectionsToRender or even in a useEffect all the way on the top, that way it breaks without having the code have to process everything up until here.